Link Component

Kathuria opened the pull request:
Added Link Component with storybook story and sample test.

Kathuria committed:
Link Component

Added Link Component with storybook story and sample test.

theajr commented on src/components/Link/Link.jsx:
```diff
@@ -0,0 +1,35 @@
+    'text'
+];
+
+const Link = ({ modifier = 'primary', disabled = false, children, className = '', onClick = () => {}, linkTag = '#', ...rest }) => {
```
@Kathuria please change ```listTag``` prop to ```href```

Kathuria replied:
@theajr : Done, please review.

theajr commented on src/components/Link/link.css:
```diff
@@ -0,0 +1,4 @@
+a.disabled {
```
Do we need this? uikit should be handling it already. Please check. If it's not handling, please do the same thing using `styled-components` because using css files will cause scope issues.

Kathuria replied:
@theajr : Yes we need it, they are not handling it. As suggested removed css and added as style. Please review

Kathuria committed:
Link Component #1

Add changes based on review comments
#1 Replaced linkTag with href
#2 Removed link.css and added as style
